opencti-platform/opencti-graphql/tests/02-integration/01-database/elasticSearch-test.js

```diff
@@ -419,7 +419,7 @@ describe('Elasticsearch pagination', () => {
     let data = await elPaginate(testContext, ADMIN_USER, READ_ENTITIES_INDICES, { search: 'malicious' });
     expect(data.edges.length).toEqual(21);
     data = await elPaginate(testContext, ADMIN_USER, READ_ENTITIES_INDICES, { search: 'with malicious' });
-    expect(data.edges.length).toEqual(45);
+    expect(data.edges.length).toEqual(46);
```

Comment:
Can you explain why?

Reply:
46 instead of 45 because of the new role Knowledge with description 'Role with knowledge capabilities'

Reply:
It should not be a new role because its just a rename of the default no?

Reply:
no, I also changed the description, and there is the word 'with' inside the description, so now the role is taking into account in the counting with the filtering 'with malicious' (presence of the world 'with' in the description).

Reply:
but the changes will be removed since we keep the default role now